core/trino-main/src/main/java/io/trino/operator/OperatorContext.java

```diff
@@ -246,6 +247,11 @@ public void setFinishedFuture(ListenableFuture<Void> finishedFuture)
         checkState(this.finishedFuture.getAndSet(requireNonNull(finishedFuture, "finishedFuture is null")) == null, "finishedFuture already set");
     }
 
+    public void recordWriterInputDataSize(long sizeInBytes)
```

Comment:
Do we need extra methods for this? I think this value is just same as `inputDataSize` for `TableWriter`